deploy/deploy_marketplace.sh

```diff
@@ -2,195 +2,72 @@
 
 set -ex
 
-RED='\033[0;31m'
-NO_COLOR='\033[0m'
-
 globalNamespace=`oc -n openshift-operator-lifecycle-manager get deployments catalog-operator -o jsonpath='{.spec.template.spec.containers[].args[1]}'`
 echo "Global Namespace: ${globalNamespace}"
+GLOBAL_NAMESPACE="${GLOBAL_NAMESPACE:-$globalNamespace}"
```

Comment:
I see you do not introduce this in this PR, but the previous line printed one value, and now you are going to use another. This would confuse the reader.